std_msgs/Header header
float64 cam_stddistance_reader          # cam_stddistance_reader reading
float64 cam_stddistance_reader_var      # cam_stddistance_reader_var reading


